### ISSUE TITLE
Replying to a message with an audio file does not work

### ISSUE THREAD

**Issue by RetroAbstract** [oxen-io/session-desktop#2573]
**Describe the bug**
Replying to a message with an audio file results in the audio message being sent as if it weren't replying to a message.

**To Reproduce**

1) Right click on a message
2) Click "Reply to message"
3) Select an audio file to reply with
4) Send
5) Notice the audio message is sent with no reference to the message you wanted to reply to

**Screenshots or Logs**

![1](https://user-images.githubusercontent.com/103272301/197337889-c9f87c40-e7ad-4bc7-89a8-5518a67fd871.jpg)

**Other information (please complete the following information):**

* Device: PC
* OS: Windows 10
* Session Version or Git commit hash: v1.10.3 (Bug has been occuring for a couple versions now)

**Comment by KeeJef** [comment]
Yeah we should wrap the audio reply in a quote if possible, also referencing this issue https://github.com/oxen-io/session-ios/issues/525 if we want to more fully redesign audio messages at the same time

**Comment by ianmacd** [comment]
The recently opened #2625 seems to be the same bug in action, albeit with an image rather than an audio file.

The title and description of this bug can probably be generalised to observe that replying only with an attachment, regardless of type, causes the sent message to be disassociated from the message it was attempting to quote.
